flow/record/adapter/elastic.py

```diff
@@ -38,14 +40,25 @@ def __init__(
         verify_certs: Union[str, bool] = True,
         http_compress: Union[str, bool] = True,
         hash_record: Union[str, bool] = False,
+        api_key: Union[str, bool] = False,
```

Comment:
Missed this in my last review, can you make this:

`api_key: Optional[str] = None`